release_notes/v3.0.0.md

```diff
@@ -0,0 +1,11 @@
+# Introducing the new Smooch Web Messenger
+
+The Web Messenger has been updated with a fresh new look and some exciting new features.
+
+![Web Messenger](http://i.imgur.com/8phlV5M.gif)
```

Comment:
I would use a static screenshot which show the top header and some bubbles. this gif I would use below